spec/features/user_features/user_delete_profile_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "rails_helper"
+
+feature "user deletes profile" do
+  let(:user) { FactoryGirl.create(:user) }
+
+
+  scenario "logged in user deletes their user profile and is redirected to login page " do
+    login_as(user, :scope => :user)
+    visit user_path(user)
+
+    click_link "Delete User Profile"
+    expect(page).to have_content ("User Profile Deleted")
+  end
```

Comment:
This is great! But it looks like we're only testing that the alert is displaying on the page, not that we've been redirected. Also, what happens if a user navigates to another user's page, I assume that they should NOT see the "Delete User Profile" link. We should have a test that confirms that.